### PR TITLE
fix ovms execution on baremetal on ubuntu

### PR DESCRIPTION
**Pull request by dtrawins**

### CODE REVIEW

**Review by rasapala — APPROVED** [review]
LGTM

File: DockerfileMakePackage [hunk 1]
```diff
@@ -45,6 +45,7 @@ RUN if [ "$ov_use_binary" == "1" ] ; then true ; else exit 0 ; fi ; find /opt/in
 RUN if [ "$ov_use_binary" == "1" ] ; then true ; else exit 0 ; fi ; find /opt/intel/openvino/deployment_tools/inference_engine/external/ -iname '*.so*' -exec cp -v {} /ovms_release/lib/ \;
 RUN if [ "$ov_use_binary" == "1" ] ; then true ; else exit 0 ; fi ; find /opt/intel/openvino/deployment_tools/ngraph/lib/ -iname '*.so*' -exec cp -v {} /ovms_release/lib/ \;
 RUN if [ "$ov_use_binary" == "1" ] ; then true ; else exit 0 ; fi ; find /opt/intel/openvino/deployment_tools/inference_engine/external/ -iname '*.so*' -exec cp -v {} /ovms_release/lib/ \;
+RUN find /usr/lib64/ -iname 'libcrypto.so.*' -exec cp -v {} /ovms_release/lib/ \;
```
Comment (kromanow94) [inline DockerfileMakePackage:48]: Is using the dot at the end of name `libcrypto.so.*` intentional? Most commonly there will be also a library with name `libcrypto.so`.

Example from my dev machine:
```bash
$ find /usr/lib64/ -iname 'libcrypto.so.*'
/usr/lib64/libcrypto.so.1.0.2o
/usr/lib64/libcrypto.so.10
/usr/lib64/libcrypto.so.1.1.1g
/usr/lib64/libcrypto.so.1.1

$ find /usr/lib64/ -iname 'libcrypto.so*'
/usr/lib64/libcrypto.so.1.0.2o
/usr/lib64/libcrypto.so.10
/usr/lib64/libcrypto.so.1.1.1g
/usr/lib64/libcrypto.so.1.1
/usr/lib64/libcrypto.so
```

Also cp will copy file content when occurs a symbolic link:
```bash
$ TMP_DIR=$(mktemp -d)

$ find /usr/lib64/ -iname 'libcrypto.so.*' -exec cp -v {} $TMP_DIR \;
'/usr/lib64/libcrypto.so.1.0.2o' -> '/tmp/tmp.jP5Py3LA9B/libcrypto.so.1.0.2o'
'/usr/lib64/libcrypto.so.10' -> '/tmp/tmp.jP5Py3LA9B/libcrypto.so.10'
'/usr/lib64/libcrypto.so.1.1.1g' -> '/tmp/tmp.jP5Py3LA9B/libcrypto.so.1.1.1g'
'/usr/lib64/libcrypto.so.1.1' -> '/tmp/tmp.jP5Py3LA9B/libcrypto.so.1.1'

$ ll $TMP_DIR
total 19M
drwxrwxr-x  2 kromanow kromanow  120 Dec 17 15:33 .
drwxrwxrwt 66 root     root     2.7K Dec 17 15:33 ..
-rwxr-xr-x  1 kromanow kromanow 5.1M Dec 17 15:33 libcrypto.so.10
-rwxr-xr-x  1 kromanow kromanow 5.1M Dec 17 15:33 libcrypto.so.1.0.2o
-rwxr-xr-x  1 kromanow kromanow 4.2M Dec 17 15:33 libcrypto.so.1.1
-rwxr-xr-x  1 kromanow kromanow 4.2M Dec 17 15:33 libcrypto.so.1.1.1g
```

To fix all above you could perform:
```bash
find /usr/lib64/ -iname 'libcrypto.so*' -exec cp -vP {} /ovms_release/lib/ \;
```

Example:
```bash
$ find /usr/lib64/ -iname 'libcrypto.so*' -exec cp -vP {} $TMP_DIR \;
'/usr/lib64/libcrypto.so.1.0.2o' -> '/tmp/tmp.jP5Py3LA9B/libcrypto.so.1.0.2o'
'/usr/lib64/libcrypto.so.10' -> '/tmp/tmp.jP5Py3LA9B/libcrypto.so.10'
'/usr/lib64/libcrypto.so.1.1.1g' -> '/tmp/tmp.jP5Py3LA9B/libcrypto.so.1.1.1g'
'/usr/lib64/libcrypto.so.1.1' -> '/tmp/tmp.jP5Py3LA9B/libcrypto.so.1.1'
'/usr/lib64/libcrypto.so' -> '/tmp/tmp.jP5Py3LA9B/libcrypto.so'

$ ll $TMP_DIR
total 9.2M
drwxrwxr-x  2 kromanow kromanow  140 Dec 17 15:37 .
drwxrwxrwt 66 root     root     2.7K Dec 17 15:35 ..
lrwxrwxrwx  1 kromanow kromanow   19 Dec 17 15:37 libcrypto.so -> libcrypto.so.1.1.1g
lrwxrwxrwx  1 kromanow kromanow   19 Dec 17 15:37 libcrypto.so.10 -> libcrypto.so.1.0.2o
-rwxr-xr-x  1 kromanow kromanow 5.1M Dec 17 15:37 libcrypto.so.1.0.2o
lrwxrwxrwx  1 kromanow kromanow   19 Dec 17 15:37 libcrypto.so.1.1 -> libcrypto.so.1.1.1g
-rwxr-xr-x  1 kromanow kromanow 4.2M Dec 17 15:37 libcrypto.so.1.1.1g
```

File: docs/host.md [hunk 2]
```diff
@@ -49,70 +49,18 @@ This guide provides step-by-step instructions to install OpenVino&trade; Model S
    cd dist/centos && tar -xzvf ovms.tar.gz
    ```
 
-## Start the Serving<a name="start-the-serving"></a>
-1. The server can be started using the command : 
+## Running the Serving
+1. The server can be started using the command in the folder, where OVMS was installed: 
 ```Bash
-/ovms/bin/ovms -help
+./ovms/bin/ovms -help
```
Comment (kromanow94) [inline docs/host.md:55]: I think there should be `--help` here:
```bash
$ docker run -ti --rm openvino/model_server -help
error parsing options: Option ‘e’ does not exist
```

**Review by kromanow94 — CHANGES_REQUESTED** [review]

File: docs/host.md (hunk 2 above)
Comment (dtrawins) [reply docs/host.md:55]: use double dash --help
docker run -ti --rm openvino/model_server --help

Comment (kromanow94) [reply docs/host.md:55]: Yes. Use double dash. In document.

**Review by kromanow94 — APPROVED** [review]

**Pull request merged**